src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -133,13 +147,20 @@ public boolean requiresReference() {
     // Intervals from command line (singleton for now)
     private List<ChromosomeInterval> intervals;
 
-    private final Map<String, FeatureReader<VariantContext>> sampleToVCFMap = new HashMap<>();
+    // List of all sample names
+    private List<String> sampleNames = new ArrayList<>();
+
+    // Linked hash map between sample names and corresponding GVCF file name
+    private Map<String, String> sampleNameToFileName = new LinkedHashMap<>();
```

Comment:
Name this `sampleNameToVcfUri`, since the String values here are produced by `IOUtils.getPath(variantFilePath).toAbsolutePath().toUri().toString();`